### DISCUSSION TITLE
Constraint that allows a group to only edit VMs and Devices with certain contacts assigned to them

### DISCUSSION THREAD

**Discussion by victoriaronja** [netbox-community/netbox#9810] [Q&A]
As the title says, I am trying to create an ObjectPermission with a constraint that will allow a group of users to only edit Virtual Machines and Devices that have a certain set of contacts assigned to them

I'm trying to make it so the group can edit only Virtual Machines and Devices that have one of or both of the contacts in the screenshot assigned to it. In this case the contact IDs are 2 (test_contact1) and 3 (test_contact2).
![2022-07-21 09_20_43-Window](https://user-images.githubusercontent.com/93338809/180154009-f37675cb-a439-4bc1-ad05-6765b7231efd.png)
Haven't quite been able to figure it out as I am rather inexperienced with JSON. I greatly appreciate any help. Thank you to everyone in advance! :)

**Comment by ryanmerolle** [comment]
I am not aware of a way to do this, most people use tenancy to achieve this.
